.github/workflows/deploy_cherry_on_main_with_k8s.yml

```
@@ -0,0 +1,52 @@
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      # 전체 레포를 루트 기준으로 체크아웃
+      - name: checkout github
+        uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)